src/lib.rs

```diff
@@ -49,6 +49,33 @@ fn rewrite_dep_path_as_relative<P: AsRef<std::path::Path>>(dep: &mut Dependency,
     }
 }
 
+// Gets the first entry out of the document as a table if it exists,
+// or gets the second one if it doesn't. If that doesn't exist
+// either, then it returns an error.
+// Borrowing rules make it hard to do this in a function,
+// so here we are.
+macro_rules! manifest_deps_toml {
```

Comment:
```suggestion
macro_rules! get_either_table_mut {
```

Probably clearer? I'm happy otherwise

Reply:
Yeah much better, thanks. I